src/Suppressor.jl

```diff
@@ -37,11 +37,14 @@ macro suppress(block)
             _logger = current_logger()
         end
 
+        # Spelling out `with_logger(f, ...)` as a try/finally so we can do `@suppress using SpammyPkg`
```

Comment:
Is this because otherwise the expression wouldn't be top-level?

Reply:
If so, can you clarify that in the comment?

Reply:
Yes, exactly, I'll add a comment